app/services/clean_unreceived_matches.rb

```diff
@@ -0,0 +1,13 @@
+class CleanUnreceivedMatches
+  def perform
+    Match.missing.each do |m|
```

Comment:
We should wrap this in a transaction block, and rollback everything on an error, and log that error. Too dangerous to run this in production like this imo. (sorry for hijacking the review)

Reply:
Sorry, I thought I had already done this. Totally forgot about it :see_no_evil:

Reply:
Oh no, noticed just now that the variable `u` is not declared :joy: , use `m` (rematch to match while you are at it)